### PR TITLE
fixed order of volunteer checklist to work for attendee volunteers

### PR DESCRIPTION
**Pull request by EliAndrewC**
We changed the default order of the volunteer checklist, but forgot to change the if-statements that determine when you can complete each step.  That meant that even though the order of the steps changed, you still had to complete them in the original order (e.g. 1-3-2-4-5).  This is now fixed.

### CODE REVIEW

File: uber/configspec.ini [hunk 1]
```diff
@@ -479,7 +479,7 @@ __many__ = string
 [[worked_status]]
 shift_unmarked = string(default="SELECT A STATUS")
 shift_worked   = string(default="This shift was worked")
-shift_unworked = string(default="Staffer didn"t show up")
+shift_unworked = string(default="Staffer didn't show up")
```
Comment (EliAndrewC) [inline uber/configspec.ini]: Unrelated typo fix.

**Comment by binary1230** [comment]
+1
